app/backend/graphql/user/user.ts

```diff
@@ -9,25 +9,45 @@ export const User = objectType({
     t.id('id')
     t.nullable.string('name')
     t.nullable.string('image')
-    t.list.field('projects', {
-      type: Project,
-      description: 'Returns the list of projects where the user is a member',
-      resolve: (user, _arguments, context) => {
-        if (!context.teamSlug) {
-          return []
+    t.nullable.field('capacityMinutes', {
+      type: 'Int',
+      description: 'Capacity hours of the use in team',
```

Comment:
```suggestion
      description: 'Capacity hours of the user in team',
```

Reply:
But that field for time not for users

Reply:
It's the users availability in minutes per week for the given team.

Reply:
I mean description is wrong, or i something don't get it ```Capacity of the user in the team```